### DISCUSSION TITLE
Best way to deploy Seq to support internal apps

### DISCUSSION THREAD

**Discussion by johnhoge** [datalust/seq-tickets#1707] [Help]
Hi,

I really like Seq and want to deploy it internally in an environment with low to moderate levels of traffic. I know it doesn't play well with IIS so I can't just install it on our web server and call it good so I'm looking for the best other choice. So far this is what I am thinking of.

Buy another Windows Server license
Set up a new VM on ESXi
Maintain and patch a new server instance
Install Seq and configure it to respond on 443 (not sure about how to do this yet...)
Add another 50GB to my backup tasks

There is a better way, right?

**Comment by liammclennan** [comment]
Hi @johnhoge ,

You can install Seq on a server with IIS so long as they listen on different ports. It will, by default, try to store log events in memory which may affect performance of other things running on the server, but you can tune this with the [cache.systemRamTarget](https://docs.datalust.co/docs/json-config#cachesystemramtarget) setting. 

Alternatively, you can avoid buying another Windows license by hosting Seq on Linux and [deploying with Docker](https://docs.datalust.co/docs/docker-deployment-overview).

**Comment by johnhoge** [comment]
Hi Liam,

I've configured the application, and set my app to send to that api key.

It does show that it is receiving data s the tiny bar graph on the end shows:

[cid:1e37f015-ff5b-4bd9-baa4-64bb14baba4e]

But if I try to look for it there is nothing.
[cid:596d2de3-dace-4fdf-9d72-467398dde0aa]

Should the "Application" be showing up on the list below?


[cid:b8ce617d-3eb4-44c7-b86d-fbb22b91cd03]
________________________________
